config/environment.js

```diff
@@ -21,6 +21,7 @@ module.exports = function (environment) {
 
     appName: 'Organisatieportaal',
     contactEmail: 'organisaties.abb@vlaanderen.be',
+    environmentName: '{{ENVIRONMENT_NAME}}', // Supported values: LOCAL, DEV, QA
```

Comment:
This [will be replaced](https://github.com/mu-semtech/static-file-service#how-to-configure-an-ember-application-at-runtime-using-environment-variables) by the actual value on the server. We just need to set the environment variable on the DEV and QA server.